packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -62,6 +66,7 @@ export default class VTable {
                                 spanAbove: rowSpan > 0,
                                 width: hasTd ? rect.width : undefined,
                                 height: hasTd ? rect.height : undefined,
+                                selected: !!td.classList.contains(TABLE_CELL_SELECTED_CLASS),

```

Comment:
I suggest we don't put selected into VCell. Because when the following result is illegal logically but is legal to this data structure:
+---------+----------+
| selected | not         |
+---------+---------+
| not        | selected |
+--------+----------+

Instead of letting VCell know the selection, we can put the selection info into VTable only. say:
 selectedRanges: [[0,0],[1,1]].

We just need to remember the selected cell indexes, then we can easily get all selected cells, or check if a cell is selected.